app/views/content_items/call_for_evidence.html.erb

```diff
@@ -0,0 +1,181 @@
+<% content_for :extra_head_content do %>
+  <%= machine_readable_metadata(
+    schema: :article
+  ) %>
+<% end %>
+
+<%= render 'shared/email_subscribe_unsubscribe_flash' %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= render 'govuk_publishing_components/components/title', @content_item.title_and_context %>
+  </div>
+  <%= render 'shared/translations' %>
+</div>
+
+<%= render 'shared/publisher_metadata_with_logo', content_item: @content_item %>
+<%= render 'shared/single_page_notification_button', content_item: @content_item %>
+<%= render 'shared/history_notice', content_item: @content_item %>
+<% if @content_item.withdrawn? %>
+  <%= render 'govuk_publishing_components/components/notice', @content_item.withdrawal_notice_component  %>
+<% end %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <% if @content_item.national_applicability.present? %>
+      <%= render "govuk_publishing_components/components/devolved_nations", {
+        national_applicability: @content_item.national_applicability,
+        type: @content_item.schema_name
+      } %>
+    <% end %>
+
+    <% if @content_item.unopened? %>
+      <% content_item_unopened = capture do %>
+        <%= t("call_for_evidence.opens") %>
+        <%= @content_item.opening_date_midnight? ? t("call_for_evidence.on") : t("call_for_evidence.at") %>
+        <time datetime="<%= @content_item.opening_date_time %>"><%= @content_item.opening_date %></time>
+      <% end %>
+      <%= render 'govuk_publishing_components/components/notice', {
+        title: t("call_for_evidence.not_open_yet"),
+        description_text: content_item_unopened,
+      } %>
+
+    <% elsif @content_item.outcome? %>
+      <%= render 'govuk_publishing_components/components/notice', title: t("call_for_evidence.closed") %>
+
+      <%= render "attachments",
+          title: t("call_for_evidence.download_outcome"),
+          legacy_pre_rendered_documents: @content_item.outcome_documents,
```

Comment:
Do we need legacy_pre_rendered_documents on this given it's a new content type? I wouldn't have thought we'd have legacy fields

Reply:
To summarise a separate discussion on Slack - @kevindew, @ollietreend and I agreed to leave this for now as all other document types published by Whitehall are still using the legacy rendering, even though they have supported the newer rendering for some time. We have added a piece of tech debt to make the switch but we need to handle it carefully because it will be very visible to users and affect a large number of documents.